src/bin/wasm2obj.rs

```diff
@@ -29,18 +29,19 @@
     )
 )]
 
-use cranelift_codegen::settings::Configurable;
-use cranelift_codegen::{isa, settings};
+use cranelift_codegen::{isa, settings, settings::Configurable};
```

Comment:
FWIF, you could actually reduce it even further as follows:

```rust
use cranelift_codegen::{isa, settings::{self, Configurable}};
```

Reply:
Nice! I was not aware of that. Thanks!